docs/pages/admin-guides/teleport-policy/crown-jewels.mdx

```diff
@@ -12,6 +12,15 @@ Crown Jewels, and how to see permission changes for these resources.
 
 ## Prerequisites
 
+<Notice type="tip">
+
+For an improved experience, we recommend using Crown Jewels in conjunction with Teleport local users or integrating with
+[Okta](../../enroll-resources/application-access/okta/okta.mdx) or [Microsoft Entra ID](./integrations/entra-id.mdx).
+This setup helps minimize the number of access path change entries generated when highly privileged ephemeral users
+log in via Teleport Auth Connectors.
```

Comment:
```suggestion
This setup helps minimize the number of access path change entries generated when ephemeral SSO users
marked as Crown Jewels log in via Teleport Auth Connectors.
```

Reply:
They don't need to be Crown Jewels. Actually the issue is much worse if the resources are Crown Jewels because it generates one event per resource they have access to